10up/action-pr-helper
# Upgrade Node.js to v20 and upgrade npm packages.

### Description of the Change
PR upgrades the Node.js version to v20 from v16 and also upgrades the following npm packages.

- `@actions/core` from `1.10.1` to `1.11.0`
- `@octokit/action` from `6.0.6` to `7.0.0`
- `@vercel/ncc` from `0.38.1` to `0.38.2`

Closes #124

### How to test the Change
- Run this Action in another repo (use this PR branch)
- Verify that no warning present there about the node version. (eg: https://github.com/iamdharmesh/action-pr-helper-test/actions/runs/11181984641)


### Changelog Entry
> Changed - Bump Node.js version from v16 to v20
> Changed - Bump `@actions/core` from `1.10.1` to `1.11.0`
> Changed - Bump `@octokit/action` from `6.0.6` to `7.0.0`
> Changed - Bump `@vercel/ncc` from `0.38.1` to `0.38.2`

### Credits
Props @dkotter @iamdharmesh 

### Checklist:
